test/GenomePositionSearchBoxTest.js

```diff
@@ -74,6 +74,19 @@ describe('Genome position search box tests', () => {
         });
       });
     });
+
+    it('Cleans up', () => {
```

Comment:
This test was missing a cleanup method but I only just noticed after reordering the tests in karma.conf.js.a